practices/practice-two/src/styles/sections/header.css

```diff
@@ -0,0 +1,251 @@
+a {
+  text-decoration: none;
+  color: #fff;
+}
+
+.banner {
+  background: linear-gradient(
+      360deg,
+      rgba(0, 0, 0, 5e-5) 0%,
+      rgba(0, 0, 0, 0.5) 100%
+    ),
+    url(../../assets/images/header-bg.jpg);
+  background-position: center;
+  background-repeat: no-repeat;
+  background-size: cover;
+}
+
+.header-wrapper {
+  font-family: var(--font-mont);
+  padding: 15px 0;
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+  box-shadow: 0 1px 0 var(--seventh-color);
+}
+
+.header-item {
+  display: flex;
+  align-items: center;
+}
+
+.header-item-logo {
+  font-weight: 400;
+  font-size: 28.75px;
+  line-height: 39px;
+  text-transform: uppercase;
+  padding-right: 10px;
+}
+
+.header-item-phone {
+  display: inline-block;
+  padding-left: 20px;
+  position: relative;
+}
+
+.header-item-phone::after {
+  position: absolute;
+  content: "";
+  width: 1px;
+  height: 20px;
+  background-color: var(--fourth-color);
+  left: 0;
+  top: 50%;
+  transform: translateY(-50%);
+}
+
+.header-item-phone-icon {
+  padding-right: 5px;
+}
+
+.c-md-navigation,
```

Comment:
c-md, how about the meaning?

Reply:
Why c?

Reply:
c is the symbol for common
md is medium

Reply:
not clearly

Reply:
navigation is ook

Reply:
I want to create two navigation bars on two types of devices, how should I name them?